ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -84,10 +96,23 @@ export class ImportCommand extends IronfishCommand {
 
     process.stdin.off('data', onData)
 
-    return JSONUtils.parse<AccountImport>(data)
+    const convertedData = ImportCommand.bech32ToJSON(data)
+    return JSONUtils.parse<AccountImport>(convertedData !== null ? convertedData : data)
   }
 
   async importTTY(): Promise<AccountImport> {
+    const userInput = await CliUx.ux.prompt('Paste the output of wallet:export', {
+      required: true,
+    })
+    try {
+      const retData = JSONUtils.parse<AccountImport>(
+        ImportCommand.bech32ToJSON(userInput) ?? userInput,
+      )
+      return retData
+    } catch (e) {
+      CliUx.ux.info('Unable to decode the account, requesting account details individually')
```

Comment:
I think it would be better if the message said
'Failed to decode the account from the provided input, please continue with the manual input below'
also we should probably swap to `CliUx.ux.error`

Reply:
Changing the message, agreed. I like `error` better, except that `error` exits, (or I don't see in the docs how to make `error` not exit).

Reply:
And by docs I guess I mean reading the `oclif/core` code

Reply:
Nevermind figured it out 😎